.github/workflows/xpu.yml

```diff
@@ -0,0 +1,24 @@
+name: xpu
+
+on:
+  push:
+    tags:
+      - ciflow/xpu/*
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'schedule' }}
+  cancel-in-progress: true
+
+jobs:
+  linux-jammy-xpu-py3_8-build:
```

Comment:
Just a note that this is only building PyTorch.  I guess that we will need to add `linux.idc.xpu` runner first before any tests can be run there.  So any changes in `test.sh` doesn't do anything yet.

Reply:
Yes, just worked with @atalman to add the xpu runners